pages/reference/base-images/base-images.md

```diff
@@ -16,12 +16,12 @@ excerpt: Docker images maintained by {{ $names.company.lower }}
 	- aarch64
 	- amd64
 	- i386
-- multiple distributions: 
+- Multiple Distributions: 
 	- [Debian](https://www.debian.org/): jessie, sid, stretch and buster
 	- [Alpine](https://alpinelinux.org/): 3.5, 3.6, 3.7, 3.8 and edge
 	- [Ubuntu](https://www.ubuntu.com/): artful, bionic, cosmic, trusty and xenial
```

Comment:
Oxford comma? :)

* jessie, sid, stretch, and buster
* 3.5, 3.6, 3.7, 3.8, and edge
* rtful, bionic, cosmic, trusty, and xenial